src/widgets/RichTextWidget.jsx

```diff
@@ -46,13 +52,7 @@ const SlateRichTextWidget = (props) => {
           className={className}
           id={id}
           name={id}
-          value={
-            typeof value === 'undefined' ||
-            typeof value.data !==
-              'undefined' /* previously this was a Draft block */
-              ? [createEmptyParagraph()]
-              : value
-          }
+          value={isValueValid(value) ? value : [createEmptyParagraph()]}
```

Comment:
`value?.data ?? `

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
I personally prefer meaningful identifiers rather than short code.

Reply:
Are you sure with your replacement?